include/nlohmann/detail/conversions/from_json.hpp

```diff
@@ -390,6 +461,14 @@ struct from_json_fn
     {
         return from_json(j, val);
     }
+
+    template<typename BasicJsonType, typename T>
```

Comment:
Can you add a brief comment for this overload of `from_json`?

Reply:
Done